examples/parser/serialized_grammar/grammar.js

```diff
@@ -0,0 +1,154 @@
+var chevrotain = require("chevrotain")
```

Comment:
This sample grammar should be written using ES6 syntax like the other examples.
I think you may have taken it from the browser examples?